query/security/ldap/src/main/java/org/codice/ddf/admin/ldap/persist/SaveLdapConfiguration.java

```diff
@@ -83,24 +90,29 @@ public SaveLdapConfiguration(ConfiguratorFactory configuratorFactory,
     @Override
     public ListField<LdapConfigurationField> performFunction() {
         Configurator configurator = configuratorFactory.getConfigurator();
+
         if (config.settingsField()
                 .useCase()
-                .equals(LOGIN) || config.settingsField()
+                .equals(AUTHENTICATION) || config.settingsField()
                 .useCase()
-                .equals(LOGIN_AND_ATTRIBUTE_STORE)) {
+                .equals(AUTHENTICATION_AND_ATTRIBUTE_STORE)) {
 
-            Map<String, Object> ldapLoginServiceProps =
-                    serviceCommons.ldapConfigurationToLdapLoginService(config);
+            Map<String, Object> ldapLoginServiceProps = new LdapServiceCommons(propertyActions,
+                    managedServiceActions).ldapConfigurationToLdapLoginService(config);
             configurator.add(featureActions.start(LdapLoginServiceProperties.LDAP_LOGIN_FEATURE));
-            configurator.add(managedServiceActions.create(LdapLoginServiceProperties.LDAP_LOGIN_MANAGED_SERVICE_FACTORY_PID,
-                    ldapLoginServiceProps));
+            if (isNotEmpty(config.pid())) {
```

Comment:
I ended up removing the pid arg and changing this to CreateLdapConfiguration. We can create a UpdateLdapConfiguration when the time comes.